presto-jmx/src/main/java/com/facebook/presto/connector/jmx/JmxTableHandle.java

```diff
@@ -33,16 +33,25 @@
     private final String connectorId;
     private final String objectName;
     private final List<JmxColumnHandle> columns;
+    private final boolean liveData;
+
+    @Deprecated
+    public JmxTableHandle(String connectorId, String objectName, List<JmxColumnHandle> columns)
+    {
+        this(connectorId, objectName, columns, true);
+    }
 
     @JsonCreator
     public JmxTableHandle(
             @JsonProperty("connectorId") String connectorId,
             @JsonProperty("objectName") String objectName,
-            @JsonProperty("columns") List<JmxColumnHandle> columns)
+            @JsonProperty("columns") List<JmxColumnHandle> columns,
+            @JsonProperty("liveData") boolean liveData)
     {
         this.connectorId = requireNonNull(connectorId, "connectorId is null");
         this.objectName = requireNonNull(objectName, "objectName is null");
         this.columns = ImmutableList.copyOf(requireNonNull(columns, "columns is null"));
```

Comment:
No that it matters much, but, IMO, "columns" is a good name. It's descriptive but not overly verbose. Saying it's a "column handle" doesn't add any more info that what could be gathered from looking at the variable name and its type.